src/Admin/Model/DoctrineRestServiceEntity.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Laminas\ApiTools\Doctrine\Admin\Model;
 
-use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Persistence\ObjectManager;
```

Comment:
Due to the change of type on `protected $objectManager`, this becomes a BC break requiring a major release :thinking:

Reply:
Same applies to other properties